Classes/Integration/HookSubscribers/DataHandlerSubscriber.php

```diff
@@ -358,6 +364,7 @@ protected function recursivelyMoveChildRecords(string $table, int $parentUid, in
     protected function getSingleRecordWithRestrictions(string $table, int $uid, string $fieldsToSelect): ?array
     {
         $queryBuilder = GeneralUtility::makeInstance(ConnectionPool::class)->getQueryBuilderForTable($table);
+	$queryBuilder->getRestrictions()->removeAll()->add(GeneralUtility::makeInstance(DeletedRestriction::class));
```

Comment:
This would break an intentional change to move soft-deleted child records when the parent is moved. Without this, undeleting a child record of a parent that was moved to another page would cause the child record to become orphaned. So this shouldn't be included - but in order to understand the reason for it, why did you add this constraint?

Reply:
I have added this contraint, because in the current development state it was not possible to delete hidden records. It endsup in an exception.